lib/classes/PluginManager.js

```diff
@@ -151,6 +152,11 @@ class PluginManager {
     this.loadPlugins(pluginsObject.modules);
   }
 
+  loadEnterprisePlugin() {
+    const depNodeModules = path.join(__dirname, '../../node_modules');
```

Comment:
Dumb question but does `path.join` recoginze `../../` and translates it to smth. os specific? Or does it just join the dirname and keep the `../../` 🤔

Reply:
it understands it. Tho.. should path.join be used for this? I feel like I ran into issues with windows using `\` but `require` doesn't care about OS, it always wants `/`. But other [things in this file](https://github.com/serverless/serverless/blob/dccec5501a83239c5c5b4775ecfd970870226c52/lib/classes/PluginManager.js#L135) are using it 🤔 I guess I'll just have to make sure to test this on windows.
```
[:foo] 1 $ cat foo.js
console.log(require('path').join(__dirname, '../../foobar'))
[:foo] $ node foo.js
/Users/dschep/code/foobar
[:foo] $ pwd
/Users/dschep/code/tmp/foo
```

Reply:
Good point. Haven't thought about that and have never experienced that before (because I don't have a Windows machine). Definitely smth. we have to look into... :+1:

Reply:
Tested on windows. works 👍